proto3-suite.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:       2.2
 name:                proto3-suite
-version:             0.8.2
+version:             0.8.3
```

Comment:
Shouldn't we go all the way to 0.9 since this is a breaking change?

Reply:
Yes